Kontent.Ai.Management/Models/Publishing/SchedulePublishAndUnpublishModel.cs

```diff
@@ -0,0 +1,36 @@
+using Newtonsoft.Json;
+using System;
+
+namespace Kontent.Ai.Management.Models.Publishing;
+
+/// <summary>
+/// Represents the schedule interval model.
+/// </summary>
+public class SchedulePublishAndUnpublishModel
+{
+    /// <summary>
+    /// Gets or sets ISO-8601 formatted date-time for scheduled publishing.
+    /// If you do not provide this property, the publishing schedule won't be updated
+    /// </summary>
+    [JsonProperty(PropertyName = "publish_scheduled_to")]
+    public DateTimeOffset? PublishScheduledTo { get; set; }
+    
+    /// <summary>
+    /// IANA time zone name used to display time offset of the scheduled publish date in the UI.
+    /// </summary>
+    [JsonProperty(PropertyName = "publish_display_timezone")]
+    public string PublishDisplayTimeZone { get; set; }
+
+    /// <summary>
+    /// Gets or sets ISO-8601 formatted date-time for scheduled unpublishing.
+    /// If you do not provide this property, the unpublishing schedule won't be updated
+    /// </summary>
+    [JsonProperty(PropertyName = "unpublish_scheduled_to")]
+    public DateTimeOffset? UnpublishScheduledTo { get; set; }
+
+    /// <summary>
+    /// IANA time zone name used to display time offset of the scheduled publish date in the UI.
+    /// </summary>
```

Comment:
according to [api specs](https://kontent.ai/learn/docs/apis/openapi/management-api-v2/#operation/schedule-publishing-and-unpublishing-of-a-language-variant), there's only one `display_timezone` property, not two separate for publishing and unpublishing

Reply:
Hi, the [api specs](https://kontent.ai/learn/docs/apis/openapi/management-api-v2/#tag/Publishing) should now be up to date.